help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -77,10 +75,11 @@ To configure details for the destination, fill in the required and optional fiel
 *  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
 *  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
 *  **[!UICONTROL Amazon Ads Advertiser ID]**: Select the ID for the target Amazon Ads account used for the destination.
+*  **[!UICONTROL Advertiser Region]**: Select the appropriate region in which your AdvertiserId is hosted. For more information on the marketplaces supported by each region, visit our [documentation](https://advertising.amazon.com/API/docs/en-us/info/api-overview#api-endpoints).
 
-Note: once selecting this Amazon Ads Advertiser ID, you will need to create a new destination to change this. If you re-authenticate the OAuth credentials and select a new Advertiser Id, your changes will not apply.
+Note: on selecting this Amazon Ads Advertiser ID, you will need to create a new destination to change this. If you re-authenticate the OAuth credentials and select a new Advertiser Id, your changes will not apply.
```

Comment:
Line 80 sounds rather confusing. What I'm understanding (correct me if I'm wrong) is that once you save the destination connection, you cannot change the Advertiser ID even if you re-authenticate. If that is correct, let's just be clear and say that:

"After saving the destination configuration, you will not be able to change the Amazon Ads Advertiser ID, even if you re-authenticate through your Amazon account. To use a different Amazon Ads Advertiser ID, you must create a new destination connection."

Also, this note applies to the Advertiser ID, so please move it just below the bullet point for Advertiser ID. Right now it's under the Advertiser Region field.

Reply:
Done.